src/nrfcloud_utils/device_credentials_installer.py

```diff
@@ -772,6 +776,20 @@ def main(in_args):
         if args.save:
             print(local_style('Saving pub key...'))
             write_file(args.path, args.fileprefix + dev_id + "_pub.pem", pub_bytes)
+    elif args.local_cert_file:
+        if not os.path.isfile(args.local_cert_file):
+            print(error_style(f'Local certificate file {args.local_cert_file} does not exist'))
+            cleanup()
+            sys.exit(11)
+
+        with open(args.local_cert_file, 'r') as f:
+            dev_bytes = f.read()
+
+        cred_if.write_credential(args.sectag, 1, dev_bytes)
```

Comment:
as Wi-Fi has CA, PUBLIC CERT, PRIATE KEY, we need an argument that takes the type too.

Reply:
Added this option